hangupsbot/plugins/image/image_memegenerator.py

```diff
@@ -30,7 +30,13 @@ def meme(bot, event, *args):
             parameters.append("robot")
 
         """public api: http://version1.api.memegenerator.net"""
-        url_api = 'http://version1.api.memegenerator.net/Instances_Search?q=' + "+".join(parameters) + '&pageIndex=0&pageSize=25'
+        url_api = 'http://version1.api.memegenerator.net//Instances_Search?q=' + "+".join(parameters) + '&pageIndex=0&pageSize=25&apiKey=e12aa6f1-ed4a-4362-aeaf-1d3281e6fa29'
```

Comment:
Whoops, you just published your private api-key.

Reply:
Wow, I definitely need some sleep... Thank you for the catch.
Any way to delete the PR?

Reply:
@renshuki The commit remains public. I suggest revoking the api-key.